### ISSUE TITLE
Journal Recruitment Status: no status sent to the AE when the invitation email differs to the profile preferred email

### ISSUE THREAD

**Issue by melisabok** [openreview/openreview-py#1422]
An AE reported that they can not see when the reviewer accepts or declines the invitation to join the TMLR reviewers group and I found a pattern: all the reviewers where they changes they preferred email don't appear in the recruitment response note. 

Check the recruitment responses: https://openreview.net/forum?id=xLdUNpcuxA for an AE: https://openreview.net/forum?id=xLdUNpcuxA&noteId=TcRFrEl1WR

**Comment by celestemartinez** [comment]
Looks like the issue here is that when we updated the recruitment process function, we did not add back `JOURNAL_REQUEST_ID` and `SUPPORT_GROUP` (we need these to get the journal request form and check if a status has been posted already). I can add this back. 

<img width="680" alt="Screen Shot 2022-10-11 at 3 55 38 PM" src="https://user-images.githubusercontent.com/32438984/195186676-4ccd81dc-c348-49df-aeb1-b1919d26edf1.png">

I don't think there should be any issues with changing the preferred email, because if the invitation was sent when a user had a profile, we use the profile ID.
